src/neuroconv/datainterfaces/ecephys/requirements.txt

```diff
@@ -1,4 +1,4 @@
-neo @ git+https://github.com/NeuralEnsemble/python-neo@50abfbce832edb1c0f8364ac34c8a6442e1a3edf
+neo>=0.11
```

Comment:
This was due to spikeinterface 0.95.1 depends on neo>=0.11

Reply:
ah yes, so we don't need neo and probeinterface here at all right?

Reply:
the same goes actually for spikeextractors>=0.9.10, SI 0.95.1. has the same pin

Reply:
Are you sure? I don't see spikeextractors listed in SI requirements - would be weird for them to do that, too, since it's in LEGACY

CI seems to agree, need to restore our deps for spikeextractors here